src/common/utility.cpp

```diff
@@ -190,6 +190,13 @@ QByteArray Utility::userAgentString()
     return re.toLatin1();
 }
 
+QByteArray Utility::friendlyUserAgentString()
+{
+    const auto pattern = QStringLiteral("%1 (Desktop Client - %2)");
+    const auto userAgent = pattern.arg(QSysInfo::machineHostName()).arg(platform());
```

Comment:
I think clazy would suggest
```suggestion
    const auto userAgent = pattern.arg(QSysInfo::machineHostName(), platform());
```

Reply:
Done